test/countBy.test.js

```diff
@@ -0,0 +1,11 @@
+import countBy from "../src/countBy.js";
+
+test("countBy works", () => {
+    expect(countBy([
+        { 'user': 'barney', 'active': true },
+        { 'user': 'betty', 'active': true },
+        { 'user': 'fred', 'active': false }
+    ], value => value.active)).toBe(
+        { 'true': 2, 'false': 1 }
+    );
+});
```

Comment:
I'd recommend using describe-it "syntax" to group the tests (in all test files), since we are going to have multiple tests for each utility function. `It()` can be used to create new tests and `describe()` can be used to further group the tests if needed. The top level describe should contain the function name 👍 

Additionally, separating the users array to a variable makes the test easier to read in my opinion 

```suggestion
describe("countBy", () => {
  it("correctly counts active users", () => {
    const users = [
      { user: "barney", active: true },
      { user: "betty", active: true },
      { user: "fred", active: false },
    ];
    expect(countBy(users, (value) => value.active)).toBe({ true: 2, false: 1 });
  });
});
```